proto/spacemesh/v1/tx_types.proto

```diff
@@ -59,7 +59,7 @@ message TransactionResultsRequest {
     // id is filter by transaction id.
     bytes id = 1;
     // address is a filter by account address, it could be principal or any affected address.
-    bytes address = 2;
+    AccountId address = 2;
```

Comment:
string

Reply:
i deliberately avoided using AccountId since it creates multiple unnecessary levels, but you changed it back